.eslintrc.js

```diff
@@ -0,0 +1,15 @@
+module.exports = {
+   extends: [
+      '@ni/eslint-config/typescript',
+      '@ni/eslint-config/typescript-requiring-type-checking',
+      'plugin:prettier/recommended'
+   ],
+   parserOptions: {
+      project: 'tsconfig.json'
+   },
+   rules: {
+      'no-console': 'off',
+      'import/no-extraneous-dependencies': 'off',
+      'no-void': 'off'
```

Comment:
I would prefer to not disable the `no-void` rule completely, and instead enable the `allowAsStatement` option.

`'no-void': ['error', { allowAsStatement: true }]`

Reply:
yes we should do that, thanks